.editorconfig

```diff
@@ -289,9 +262,6 @@ dotnet_diagnostic.IDE2001.severity = warning
 # csharp_style_allow_blank_lines_between_consecutive_braces_experimental
 dotnet_diagnostic.IDE2002.severity = warning
 
-# dotnet_style_allow_statement_immediately_after_block_experimental
-dotnet_diagnostic.IDE2003.severity = warning
```

Comment:
This has no effect 😕 

The corresponding code-style option default value is true which means "disable the analyzer":

https://github.com/dotnet/roslyn/blob/85c8cf51143af5ac9b782ff959aa286b4a6e2e38/src/Analyzers/Core/Analyzers/NewLines/ConsecutiveStatementPlacement/AbstractConsecutiveStatementPlacementDiagnosticAnalyzer.cs#L41-L43

I guess it's not intentional, @CyrusNajmabadi ?